style.css

```diff
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  overflow: hidden;
+}
+
+.main {
+  background: url("./images/header-shapes\ mobile.svg") no-repeat center center;
+  height: 410px;
+  width: 100%;
+  padding: 10px;
+  justify-content: center;
+  margin-left: 30px;
+}
```

Comment:
- [ ] Your background is not displaying so kindly check this and make changes to it as you are using width and height and it is creating a problem on the background.
### your background
![back](https://user-images.githubusercontent.com/81727988/170713083-c2122913-56e1-4268-af12-c0a80ade981f.png)